netlify.toml

```diff
@@ -4,15 +4,27 @@
 
 # Temporary header config until Netlify supports setting `headers` on `next.config.js`
 # https://github.com/netlify/netlify-plugin-nextjs/issues/150
+
+[[headers]]
+for = "/admin*"
+  [headers.values]
+     X-Frame-Options = "DENY"
+    X-XSS-Protection = "1; mode=block"
+    X-Content-Type-Options = "nosniff"
+    Referrer-Policy = "same-origin"
+    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com *.google-analytics.com; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
+    Permissions-Policy = "camera=(), microphone=(), geolocation=(), interest-cohort=()"
+
 [[headers]]
   for = "/*"
   [headers.values]
     X-Frame-Options = "DENY"
     X-XSS-Protection = "1; mode=block"
     X-Content-Type-Options = "nosniff"
     Referrer-Policy = "same-origin"
-    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
+    Content-Security-Policy = "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' *.googletagmanager.com *.netlify.com unpkg.com analytics.google.com *.google-analytics.com; img-src 'self' blob: data: https:; frame-ancestors 'none'; "
```

Comment:
I think here, we don't need to allow `*.netlify.com unpkg.com`

Reply:
But it can be work later

Reply:
Let's see the result first.

Reply:
ah yap, since it's only needed in admin page

Reply:
Let's keep this to be work on the next PR